xbmc/GUIInfoManager.cpp

```diff
@@ -497,8 +487,6 @@ const infomap player_labels[] =  {{ "hasmedia",         PLAYER_HAS_MEDIA },
                                   { "forwarding8x",     PLAYER_FORWARDING_8x },
                                   { "forwarding16x",    PLAYER_FORWARDING_16x },
                                   { "forwarding32x",    PLAYER_FORWARDING_32x },
-                                  { "canrecord",        PLAYER_CAN_RECORD },
-                                  { "recording",        PLAYER_RECORDING },
```

Comment:
The defines for `PLAYER_CAN_RECORD` and `PLAYER_RECORDING` (https://github.com/xbmc/xbmc/blob/master/xbmc/guiinfo/GUIInfoLabels.h#L44) should be removed as well.

Reply:
ok